packages/editor/src/css/style_mosaico_tools.less

```diff
@@ -760,6 +760,20 @@
     }
   }
 
+  .small-modal .modal {
+    max-height: 260px;
+  }
+  .error-button {
+    background-color: @error-color;
+  }
+
+  .small-text {
+    font-size: 1.2em;
```

Comment:
Is this text really smaller than the default one ? because 1.2em should increase the size by 20% if I'm correct, and it's not smaller in fact ^^

Reply:
maybe the name of the class is not good enough, but what i wanted is not really a small text, just a text size smaller than the title of the modal to make it looks good and clear ! I will change the name of the css class ! thank you for this feedback